### PR TITLE
setup project (SDKCF-3516)

### PR DESCRIPTION
**Pull request by imaginaris**

### CODE REVIEW

File: RSignatureVerifier.podspec [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  s.name         = "RSignatureVerifier"
+  s.version      = "0.1.0"
+  s.authors      = "Rakuten Ecosystem Mobile"
+  s.summary      = "Rakuten's Signature Verifier module."
+  s.homepage     = "https://github.com/rakutentech/ios-signatureverifier"
+  s.license      = { :type => 'MIT', :file => 'LICENSE' }
+  s.source       = { :git => "https://github.com/rakutentech/ios-signatureverifier.git", :tag => s.version.to_s }
+  s.platform     = :ios, '11.0'
+  s.swift_version = '5.3'
+  s.documentation_url = "https://rakutentech.github.io/ios-remote-config/"
```
Comment (donnie-jp) [inline RSignatureVerifier.podspec]: "remote-config" -> "signatureverifier"

File: SampleAppTests/SampleAppTests.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
+//
+//  SampleAppTests.swift
+//  SampleAppTests
+//
+//  Created by Kalinowski, Michal on 31/03/2021.
+//  Copyright © 2021 Rakuten Group, Inc. All rights reserved.
+//
```
Comment (donnie-jp) [inline SampleAppTests/SampleAppTests.swift]: No file headers please

File: RSignatureVerifier/Verifier.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
+internal class Verifier {
```
Comment (donnie-jp) [inline RSignatureVerifier/Verifier.swift:1]: I didn't expect any functionality to be added to the project yet.
I thought that'd be in a separate PR but if it's included and the tests pass then I think it's OK.

**Review by donnie-jp — CHANGES_REQUESTED** [review]
I left a couple of minor inline comments.

This is fine for merging as long as the bitrise CI status is reported here. I see the bitrise build passed so I'm not sure why the PR hasn't been updated with the status.

File: Cakefile [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# https://github.com/jcampbell05/xcake
+# http://www.rubydoc.info/github/jcampbell05/xcake/master/file/docs/Cakefile.md
```
Comment (donnie-jp) [inline Cakefile:2]: Did you use xcake to generate the project or just copy/modify the remote-config repo?

Comment (imaginaris) [reply Cakefile:2]: I used xcake. Although I modified schemes to react to "Cmd+U" command.

**Comment by imaginaris** [comment]
> I left a couple of minor inline comments.
> 
> This is fine for merging as long as the bitrise CI status is reported here. I see the bitrise build passed so I'm not sure why the PR hasn't been updated with the status.

It's complicated :)
The Bitrise couldn't see my branch in my forked repo so I pushed it to main repo just to create the project.
Plus it takes the config from master/main branch. (I temporarily modified the main workflow so we can see the actual build status. I'll revert that after this PR is merged)

File: RSignatureVerifier.podspec (hunk 1 above)
Comment (imaginaris) [reply RSignatureVerifier.podspec]: Fixed

File: SampleAppTests/SampleAppTests.swift (hunk 2 above)
Comment (imaginaris) [reply SampleAppTests/SampleAppTests.swift]: Fixed

**Review by donnie-jp — APPROVED** [review]
